From 00149d0f3775c81c541f48f83860a45998e34c66 Mon Sep 17 00:00:00 2001
From: Juan RP <xtraeme@voidlinux.eu>
Date: Sat, 20 Jun 2015 09:48:21 +0200
Subject: [PATCH] xbps_transaction_prepare: if ENOSPC, keep the transaction
 dictionary.

... we need this in xbps-install to print required/free space on rootfs.

Close #101
---
 NEWS                         | 4 ++++
 lib/transaction_dictionary.c | 2 --
 2 files changed, 4 insertions(+), 2 deletions(-)

--- lib/transaction_dictionary.c
+++ lib/transaction_dictionary.c
@@ -344,8 +344,6 @@ xbps_transaction_prepare(struct xbps_handle *xhp)
 	 * and removed to the transaction dictionary.
 	 */
 	if ((rv = compute_transaction_stats(xhp)) != 0) {
-		xbps_object_release(xhp->transd);
-		xhp->transd = NULL;
 		return rv;
 	}
 	/*
-- 
2.4.4

